src/server/session.ts

```diff
@@ -1411,7 +1438,12 @@ export class Session<TMessage = string> implements EventSender {
                 return next.immediate("regionSemanticCheck", () => {
                     const scriptInfo = this.projectService.getScriptInfoForNormalizedPath(fileName);
                     if (scriptInfo) {
-                        this.regionSemanticCheck(fileName, project, ranges.map(range => this.getRange({ file: fileName, ...range }, scriptInfo)));
+                        this.regionSemanticCheck(
+                            fileName,
+                            project,
+                            ranges.map(range => this.getRange({ file: fileName, ...range }, scriptInfo)),

```

Comment:
Call me superstitious but

```suggestion
                            ranges.map(range => this.getRange({ ...range, file: fileName }, scriptInfo)),
```